docs/qute/TemplateSupport.md

```diff
@@ -1,105 +1,105 @@
-
-![Highlight Corresponding Tag](images/QuteTemplateSectionHighlighting.gif)
-
-## Expression support
-
-A Qute expression is declared inside `{|}` or inside a section (ex : `{#for item in |}`).
-
-### Completion for Java data model
-
-Completion is available for allowed Java `fields`, `methods` and `virtual methods` (coming from the Qute core like `orEmpty` or coming from @TemplateExtension):
-
-![Completion for Java data model](images/QuteTemplateCompletionForJavaDataModel.gif)
-
-### Validation for Java data model
-
-Validation highlights unknown Java fields or methods as an error:
-
-![Validation for Java data model](images/QuteTemplateValidationForJavaDataModel.png)
-
-### Hover for Java data model
-
-When you hover over an object, property or method or virtual method (for `@TemplateExtension`), you can see the Java type:
-
-![Hover for Java data model](images/QuteTemplateHoverForJavaDataModel.gif)
-
-When you hover over a [Built-in Resolvers](https://quarkus.io/guides/qute-reference#built-in-resolvers) you can see the relevant documentation, with a link to the [Qute reference guide](https://quarkus.io/guides/qute-reference):
-
-![Hover for Built-in Resolvers](images/QuteTemplateHoverForBuildInResolver.png)
-
-### Jump to Java field, method definition
-
-`Ctrl+Click` on a property or method to jump to the definition of the matching Java field, method or virtual method (for `@TemplateExtension`):
-
-![Definition for Java data model](images/QuteTemplateDefinitionForJavaDataModel.gif)
-
-### Jump to the declared object
-
-A variable can be declared inside a Qute template (with parameter declaration, or #let section) or declared from the Java side.
-
-`Ctrl+Click` on an object part jumps to the declared object in the Qute template (with parameter declaration, or #let section) or to the Java code (method parameter or data method invocation):
-
-![Definition for Java data model](images/QuteTemplateDefinitionForObject.gif)
-
-## Quarkus integration for Qute
-
-When the template is used in a Quarkus application and it is bound with a Java Template field or a method annotated with `@CheckedTemplate`, you can see the expected model as CodeLens:
-
-![CodeLens for Java data model](images/QuteTemplateCodeLensJavaDataModel.png)
-
-In this case, you don't need to declare parameter declaration to benefit from Qute completion, validation, etc. of the Java data model.
-
-CodeLens are clickable to jump to the proper Java type, method or field.
\ No newline at end of file
+# Qute Template support
+
+Qute support provides `completions`, `validation`, `hover`, `definition`, `highlight`, `linked editing` in Qute templates.
+
+To benefit from this support, HTML, Text, YAML and JSON files in `**/resources/templates/**` are automatically opened as Qute templates. You can disable that behavior by setting the `quarkus.tools.propertiesLanguageMismatch` property to `ignore`, or `prompt`, if you prefer being asked to change the type when opening the file. 
```

Comment:
...to `ignore`, or `prompt` if you prefer... (no comma)

Reply:
The "or" begins an independent clause (since it's separate from the `ignore` when separated by a comma), which is also proceeded by a description. If I were to describe something, I wouldn't put a comma, (i.e. `prompt` is the option to ask the user if they prefer being asked to change the type when opening the file).

Reply:
Ah right. the description describes  `prompt` and not the option in general. Ok, then sure, the comma shouldn't be there :P

Reply:
Sorry if this is nit-picky, just a small adjustment

Reply:
geeez ;-)

Reply:
Fixed